src/test/gcp/cloudscheduler.spec.ts

```diff
@@ -0,0 +1,97 @@
+import * as _ from "lodash";
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as nock from "nock";
+import * as api from "../../api";
+
+import { cloudscheduler } from "../../gcp/";
+
+const VERSION = "v1beta1";
+
+const testJob = {
+  name: "projects/test-project/locations/us-east1/jobs/test",
+  schedule: "every 5 minutes",
+  timeZone: "America/Los_Angeles",
+  httpTarget: {
+    uri: "https://afakeone.come",
+    httpMethod: "POST",
+  },
+  retryConfig: {},
+};
+describe("cloudscheduler", () => {
+  let sandbox: sinon.SinonSandbox;
+
+  describe("createOrUpdateJob", () => {
+    beforeEach(() => {
+      sandbox = sinon.createSandbox();
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+      nock.cleanAll();
+    });
+
+    it("should create a job if none exists", async () => {
+      /*tslint:disable*/
```

Comment:
Why is it necessary to disable the linter?

Reply:
Had it there during dev so i could use console log, and I forgot to get rid of it, whoopsies